x/mint/types/params.go

```diff
@@ -50,7 +50,7 @@ func NewParams(
 func DefaultParams() Params {
 	return Params{
 		MintDenom:               sdk.DefaultBondDenom,
-		GenesisEpochProvisions:  sdk.NewDec(6_700_000).QuoInt64(365),
+		GenesisEpochProvisions:  sdk.NewDec(6_700_000_000_000),
```

Comment:
@riley-stride did you make this change?

Reply:
iirc you mentioned this param yesterday, not sure what it does

Reply:
addressed this